config/options.go

```diff
@@ -30,6 +36,11 @@ import (
 // DisableHeaderKey is the key used to check whether to disable setting header
 const DisableHeaderKey = "disable"
 
+const (
+	idpCustomScopesDocLink = "https://www.pomerium.io/reference/#identity-provider-scopes"
+	idpCustomScopesWarnMsg = "config: using custom scopes may result in undefined behavior, see: " + idpCustomScopesDocLink
```

Comment:
nit++ but I think we should just have one variable here. `warnCustomScopes` and just use multi-line breaks so it's readable. Similar to `ErrXYZ`. Thoughts?

Reply:
Or, just return the full warning string without setting either the message or the link to a variable if its not going to be used elsewhere.

Reply:
My intentional is that the dock link can be changed more frequently than the error message. so put it in a constant so we can grep or edit easier. Multi-line breaks seems not appropriate because it causes the message divided into 2 lines.

In all case, the compiler can optimize that for us, whether use separated constants or big one.